docs/publishing.md

```diff
@@ -13,8 +13,9 @@ Marketplace](https://marketplace.visualstudio.com/vscode).
 
 For more information about publishing take a look at 
 
-* [Publishing VS Code Extensions](https://code.visualstudio.com/docs/extensions/publish-extension)
-* [Managing Extensions](https://code.visualstudio.com/docs/editor/extension-gallery)
+* [Publishing VS Code Extensions][publish_vscode_ext]
+* [Managing
```

Comment:
@lcampos - If you look at https://github.com/forcedotcom/salesforcedx-vscode/blob/16d6f0634a0b107da8fa7705c16787c4bcae343a/docs/publishing.md it renders the file weirdly. Does GitHub not like this style of Markdown? Or is it because you have a new line between 17 and 18?

Reply:
I think I messed up the ``` alignment for the s3 command after executing the rewrap, let me fix it.